src/components/profile/add-place/add-place.js

```diff
@@ -1,36 +1,106 @@
 import React, { Component } from "react";
+import RegionsService from '../../../services/regions-service';
+import Api from '../../../services/places-service';
 import "./add-place.scss";
 
 export default class AddPlace extends Component {
+  constructor() {
+    super();
+    this.regionsService = new RegionsService();
+    this.state = {
+      regionsList: [],
+      title: '',
+      description: '',
+      isModerated: false,
+      regionId: ''
+    }
+  }
+
+  componentDidMount() {
+    this.getRegionsList();
+  }
+
+  getRegionsList = () => {
+    this.regionsService.getRegionsList().then((regions) => {
+      this.setState({ regionsList: regions })
+    }).catch((error) => {
+      console.log(error)
+    })
+  }
+
+  handleChange = (event) => {
+    const name = event.target.name;
+    this.setState({ [name]: event.target.value });
+  }
+
+  addPlace = (placeInfo) => {
+    const newPlaceInfo = {
+      isModerated: false,
+      title: placeInfo.title,
+      description: placeInfo.description,
+      regionId: (() => {
+        let region;
+        placeInfo.regionsList.find(city => {
+          if (city.name.trim() === placeInfo.region.trim()) {
+            return region = city
+          }
+        }
+        )
```

Comment:
Suggestion:
```
const region = placeInfo.regionsList.find(city => city.name.trim() === placeInfo.region.trim());
```

Reply:
you cant check this task without request on server, unfortunatelly I cant create it till Dima`s request is opened